api/src/wfl/module/covid.clj

```diff
@@ -625,13 +623,12 @@
   Return EXECUTOR."
   [source executor]
   (if-let [snapshot (peek-queue! source)]
-    (let [reference        (from-source executor snapshot)
-          updated-executor (update-method-configuration! executor reference)
-          submission       (create-submission! updated-executor reference)]
-      (write-workflows! updated-executor reference submission)
-      (pop-queue! source)
-      (update-terra-workflow-statuses! updated-executor))
-    (update-terra-workflow-statuses! executor)))
+    (let [reference  (from-source executor snapshot)
+          submission (create-submission! executor reference)]
+      (write-workflows! executor reference submission)
+      (pop-queue! source)))
+  (update-terra-workflow-statuses! executor)
+  executor)
```

Comment:
when-let instead of if-let?
Passing an input through to the output always makes me think.

Reply:
>when-let instead of if-let?
Thanks, good suggestion!

>Passing an input through to the output always makes me think.
Yeah, this seemed a bit funny to me too.  We may elect to lightly refactor `update-covid-workload` to avoid having to do this.

Reply:
Switched to `when-let`.